phpunit.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you commit a travis.yml file and a composer.json with PHPUnit? I think we can load the composer libs within Travis and get the same functionality